### ISSUE TITLE
Installing “dart-tools@0.9.10” failed. (Windows 8.1)

### ISSUE THREAD

**Issue by mahonnaise** [radicaled/dart-tools#51]
With Atom 0.201.0 on Windows 8.1, installation of dart-tools 0.9.10 fails with the following message:

```
> runas@2.0.0 install C:\Users\K\AppData\Local\Temp\apm-install-dir-115423-3028-1wenwn6\node_modules\dart-tools\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\K\AppData\Local\Temp\apm-install-dir-115423-3028-1wenwn6\node_modules\dart-tools\node_modules\pathwatcher\node_modules\runas>node "C:\Users\K\AppData\Local\atom\app-0.201.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\K\\AppData\\Local\\atom\\app-0.201.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\K\\AppData\\Local\\atom\\app-0.201.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\K\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\K\\.atom\\.apmrc" "install" "C:\\Users\\K\\AppData\\Local\\Temp\\d-115423-3028-1tbh5ov\\package.tgz" "--target=0.22.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

**Comment by radicaled** [comment]
This seems like a bug in one of our dependencies: https://github.com/atom/node-runas/issues/9

We only use PathWatcher in one location... I'm going to see if there's an alternate package we can use, hopefully something baked into node/io.js.

**Comment by radicaled** [comment]
Resolved in 0.9.11.

**Comment by mahonnaise** [comment]
Works now. Thanks. :)
